core/src/test/java/org/infinispan/remoting/transport/jgroups/JGroupsTransportTest.java

```diff
@@ -50,7 +50,7 @@ public void testSynchronousIgnoreLeaversInvocationToNonMembers() throws Exceptio
 
       JGroupsTransport transport = (JGroupsTransport) manager(0).getTransport();
       long initialMessages = transport.getChannel().getSentMessages();
-      ReplicableCommand command = new ClusteredGetCommand("key", CACHE_NAME, 0);
+      ReplicableCommand command = new ClusteredGetCommand("key", CACHE_NAME, TestingUtil.getSegmentForKey("key", cache(0)), 0);
```

Comment:
The segment shouldn't matter, we're sending the command to a different cache anyway.